packages/metaconfig/test/fixtures/v0.16.0/platformatic.db.schema.json

```diff
@@ -548,7 +548,14 @@
               ]
             },
             "hotReload": {
-              "type": "boolean",
+              "anyOf": [
+                {
+                  "type": "boolean"
+                },
+                {
+                  "type": "string"
+                }
+              ],
```

Comment:
This should not be needed, why did you change it?

Reply:
sorry, I thought this was a test to check for types of hotReload. I will edit this out.